src/webui/webui.h

```diff
@@ -52,7 +54,9 @@ class WebUI final : public ApplicationComponent<QObject>
     Q_DISABLE_COPY_MOVE(WebUI)
 
 public:
-    explicit WebUI(IApplication *app);
+    static constexpr bool ALLOW_TEMP_PASSWORD = true;
```

Comment:
Why? Doesn't this just take unnecessary memory space?
IMO it would be preferable to just make the option strongly typed instead. eg something like this:
```c++
enum class TEMP_PWD : char {
  DISABLED: 0,
  ENABLED: 1
}

explicit WebUI(IApplication *app, TEMP_PWD isTempPasswordAllowed = TEMP_PWD::DISABLED);
```

But I would vote for staying with `bool` type and not having a variable defined for a special value. The constructor's parameter name is sufficient.

Reply:
> The constructor's parameter name is sufficient.

It is enough inside the class itself. I care about the place of its use. Compare the following:
```c++
// What is the difference between the following instances?
// You need to at least look at the class declaration.
new WebUI(this, true);
new WebUI(this, false);

// The following looks more self-descriptive, doesn't it?
new WebUI(this);
new WebUI(this, WebUI::ALLOW_TEMP_PASSWORD);
```

Of course I can use `enum` instead.

Reply:
>Of course I can use enum instead.

Done.